src/platform/platform_win.cpp

```diff
@@ -1064,7 +1054,9 @@ long long mp::platform::Platform::get_total_ram() const
     return status.ullTotalPhys;
 }
 
-std::filesystem::path mp::platform::Platform::get_root_cert_path() const
+std::filesystem::path mp::platform::Platform::get_root_cert_dir() const
 {
-    return multipass_final_storage_location() / "data" / "certificates" / "multipass_root_cert.pem";
+    // C:\ProgramData is stable and user agnostic
+    static const std::filesystem::path base_dir = "C:\\ProgramData";
```

Comment:
Drive label C:/ is not guaranteed. Perhaps we can use a Windows API function for it?

Reply:
Using `CSIDL_COMMON_APPDATA` since ["This folder is used for application data that is not user specific"](https://learn.microsoft.com/en-us/windows/win32/shell/csidl) and isn't overly restrictive so it fits our needs. It isn't guaranteed to be `ProgramData` but the other guarantees are more important